### PR TITLE
Desktop promo subheading

### PR DESCRIPTION
**Pull request by mstriemer**
screenshot 2015-01-08 16 45 46
> 1024px

screenshot 2015-01-08 16 39 40
> 1280px

### CODE REVIEW

**Comment by ngokevin** [comment]
i think it could use some more padding on top of the ```hr``` to match the padding below (as well as the padding between the promo blocks)

File: src/media/css/desktop-promo.styl [hunk 1]
```diff
@@ -87,6 +87,20 @@ for $name in games productivity box soundcloud hexrace {
...
+
+.desktop-promo-separator-text {
+    font-size: 20px;
+    font-weight: 100;
+    margin: 20px 0;
+    text-align: center;
+}
+
 // Offsets
 // IMPORTANT: This section is duplicated because we are using an outdated
```
Comment (ngokevin) [inline src/media/css/desktop-promo.styl]: i've tried to upgrade stylus before, i remember it wasn't trivial :(

File: src/templates/feed.html [hunk 2]
```diff
@@ -4,6 +4,9 @@
 
 {% if showPromo %}
   {% include 'desktop-promo.html' %}
+
+  <div class="desktop-promo-separator"></div>
+  <div class="desktop-promo-separator-text">{{ _('The Best of Marketplace') }}</div>
```
Comment (ngokevin) [inline src/templates/feed.html]: ```p```?

**Comment by ngokevin** [comment]
r+wc

**Comment by mstriemer** [comment]
The padding is all 20px in my screenshots.

**Comment by ngokevin** [comment]
weird, it looks a little bit less above than below on the shots

**Comment by mstriemer** [comment]
I think it has something to do with that font. Everything always seems to be off a bit vertically with it.

**Pull request merged**